src/Middleware/RequestThrottling/src/Internal/LoggerExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.RequestThrottling.Internal
+{
+    internal static class LoggerExtensions
+    {
+        private static readonly Action<ILogger, Exception> _requestEnqueued;
+        private static readonly Action<ILogger, Exception> _requestDequeued;
+
+        static LoggerExtensions()
+        {
+            _requestEnqueued = LoggerMessage.Define(
+                logLevel: LogLevel.Debug,
+                eventId: new EventId(1, "Request Enqueued"),
+                formatString: "Server is busy; queuing request in middleware."
+                );
+            _requestDequeued = LoggerMessage.Define(
+                logLevel: LogLevel.Debug,
+                eventId: new EventId(2, "Request Dequeued"),
+                formatString: "Space availible on server; request is leaving queue."
```

Comment:
I think we normally just define these inline with the declaration, otherwise these classes get big pretty quickly.

See [SignalR's DefaultHubDispatcher.Log.cs](https://github.com/aspnet/AspNetCore/blob/01a87989364dea1b18743a9fe3757eccc220b2ca/src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.Log.cs ) for an example.

Reply:
Inlined the stuff. I was copying another logging file :(